cluster/senza-definition.yaml

```diff
@@ -185,9 +185,7 @@ Resources:
       - PolicyDocument:
           Statement:
           - {Action: 'ec2:Describe*', Effect: Allow, Resource: '*'}
-          - {Action: 'autoscaling:Describe*', Effect: Allow, Resource: '*'}
-          - {Action: 'autoscaling:SetDesiredCapacity', Effect: Allow, Resource: '*'}
-          - {Action: 'autoscaling:TerminateInstanceInAutoScalingGroup', Effect: Allow, Resource: '*'}
+          - {Action: 'autoscaling:*', Effect: Allow, Resource: '*'}
```

Comment:
Maybe we should comment out the old actions instead of deleting them, just so it's easier to re-add later on.